articles/prettier-setup/index.md

```diff
@@ -0,0 +1,188 @@
+---
+layout: engineering-education
+status: publish
+published: true
+url: /engineering-education/prettier-setup/
+title: How to setup code formatting for Node.js apps using Prettier and ESLint
+description: How to setup code formatting for Node.js apps using Prettier and ESLint
+author: aman-saxena
+date: 2020-10-19T00:00:00-05:00
+topics: []
+excerpt_separator: <!--more-->
+images:
+
+  - url: /engineering-education/prettier-setup/hero.jpg
+    alt: image example
+
+---
+In open-source development, handling a large number of files across every development stage is quite a tedious task. These stages consist of enormous chunks of code that not only require a great amount of time needed for formatting but also differ in style with different individuals in the team.
+<!--more-->
+### Setting up code formatting
+Have you ever wondered how to cut short formatting time with just one-click? Any alternatives to prevent this monotonous task?
+
+Getting rid of the hassles of writing clean code and maintaining consistency across a project on specific code style guidelines, **Prettier** is one of the most prominent tools out there in which you can bank upon! With Prettier and githook integration, you the code is formatted within a single-click. Prettier, along with the support of **ESLint**, is a deadly combination that increases productivity manifolds.
+
+ This guide includes all the extensive information to get used to the code formatting style in the open source circuit. Let's get started by answering in-detail all the why(s), what(s) & how(s)!
```

Comment:
might be helpful to explain what "code formatting style" is ? or add a link to a definition. 🚀